user_guide_src/source/general/helpers.rst

```diff
@@ -92,7 +92,7 @@ use, you'll call it the way you would a standard PHP function.
 For example, to create a link using the ``anchor()`` function in one of
 your view files you would do this::
 
-    <?php echo anchor('blog/comments', 'Click Here');?>
+    <?php echo anchor('blog/comments', 'Click Here') ?>
```

Comment:
```suggestion
    <?= anchor('blog/comments', 'Click Here') ?>
```

Reply:
On it is written, standard coding.

Reply:
I don't get what you mean. You used the short echo tag for `esc` , `set_value`, etc., but not for `anchor`??

Reply:
updated